### ISSUE TITLE
Missing import

### ISSUE THREAD

**Issue by phptek** [vulcandigital/silverstripe-pagefeedback#11]
The class `PageFeedbackControllerExtension` calls `FormSpamProtectionExtension` but never imports it, meaning that guards that permit a Spam Protection Protector implementation never actually pass.

```
use SilverStripe\SpamProtection\Extension\FormSpamProtectionExtension;
```

...fixes it.

As an aside, the `use SilverStripe\Core\Flushable;` import isn't used.

**Comment by zanderwar** [comment]
Hey @phptek would you mind submitting your fixes as a pull request please? <3

**Comment by phptek** [comment]
See #13 :-)

**Comment by zanderwar** [comment]
Thanks @phptek merged and 2.0.1 released
